web/src/message_edit.js

```diff
@@ -471,8 +471,6 @@ function edit_message($row, raw_content) {
     currently_editing_messages.set(message.id, edit_obj);
     message_lists.current.show_edit_message($row, edit_obj);
 
-    $form.on("keydown", handle_message_row_edit_keydown);
-
     $form
         .find(".message-edit-feature-group .video_link")
         .toggle(compose_call.compute_show_video_chat_button());
```

Comment:
Shouldn't these other elements have the same reasoning apply to them?

Reply:
Oh, it looks like the $form element we store for the currently editing messages isn't a good idea. So, we will need to just store the list of currently editing message ids and restore them on rerender via a full edit mechanism. I will open a new PR for this. The change still looks good though.

Reply:
I guess so; we should be able to just use the old strategy if this logic was called whenever we're presenting a form, whether via rerendering or opening message edit in the first place.